src/tests/fnmatch/fnmatch_test.c

```diff
@@ -0,0 +1,83 @@
+/**
+ * @file
+ * @brief Test for fnmatch function.
+ *
+ * @date 19.12.18
+ * @author Filipp Chubukov
+ */
+
+#include <fnmatch.h>
+#include <embox/test.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <errno.h>
+
+#define TEST_STRING "abc"
+
+EMBOX_TEST_SUITE("posix/fnmatch test");
+
+TEST_CASE("Wrong pattern") {
+	char test_string[] = TEST_STRING;
+	char pattern[] = "a[v]gg[g[a][";
+	
+	test_assert(fnmatch(pattern, test_string, 0) == EINVAL);
+}
+
+TEST_CASE("String fits a (*) and (?) patterns") {
```

Comment:
Using parentheses can be misleading in this context (as sometimes they are used as special symbols for regex), so it's better to use `'*'` and so on.

Reply:
done